AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -441,8 +442,14 @@ - (void)applyEnabledState:(BOOL)isEnabled {
 #pragma mark - MSAuthTokenContextDelegate
 
 - (void)authTokenContext:(MSAuthTokenContext *)__unused authTokenContext didUpdateAccountIdWithAuthToken:(NSString *)authToken {
-  if (!authToken) {
+
+  // TODO: consume the unique account id once provided in authTokenContext.
+  NSString *uniqueAccountId = @"unique-account-id";
+  if (authToken) {
```

Comment:
There is something I don't understand, the delete takes a *partition* but the create takes a *table name*. Yet the same value is used for both APIs. In the Document Storage I can see that the create actually creates a table from the partition name but the delete will only delete tables with name from either `kMSAppDocumentTableName` constant or `kMSUserDocumentTableName` constant. Does identified users have their own table from their ID or are they always using a table with name: `kMSUserDocumentTableName`? Either way it seems that these operations are specific to signed in users only, there is probably some simplification to do here.

Reply:
Yes thats why there is a TODO on top. We want to create a table per logged in user but have the account id in the table name as well and we dont have a way of getting it in this delegate right now. So to get past it for now we are using a constant name. But once we get the account id we have a better flow .